Core/Core/Logging/Setup.cs

```diff
@@ -74,4 +75,12 @@ static Setup()
       Analytics.AddConnectorToProfile(account.GetHashedEmail(), hostApplication);
     }
   }
+
+  [Obsolete("Use " + nameof(Mutex))]
+  [SuppressMessage("Style", "IDE1006:Naming Styles")]
+  public static Mutex mutex
+  {
+    get => Mutex;
+    set => Mutex = value;
+  }
```

Comment:
This is new but also obsolete? I'm assuming it's coming from some other file where it was moved, but I must have missed it.

Reply:
![image](https://github.com/specklesystems/speckle-sharp/assets/45512892/d61df51a-aca2-4e53-9ecf-fd54d2e7fca9)

This was the real change, This prop was simply to not break Manager, and give us chance to gracefully transition.